frontend/src/metabase/dashboard/hooks/use-embed-theme.ts

```diff
@@ -30,10 +28,8 @@ export const useEmbedTheme = (
   const hasNightModeToggle = theme !== "transparent";
 
   useEffect(() => {
-    if (themeName) {
-      setTheme(themeName);
-    }
-  }, [themeName, setTheme]);
+    setTheme("light");
```

Comment:
Should we have a condition here - if the `theme` is `null` we set it to `light`, otherwise leave it?

Reply:
I'm always cautious when I see a `useEffect` that doesn't depend on a condition, that's all

Reply:
Maybe we don't even need this at all. The default value is already `"light"`, so `theme` will never be `null`, it can only be what users have passed through the URL parameter.
https://github.com/metabase/metabase/blob/fc0c4a9a7aef35fc01597b7abea6dd454952eb6e/frontend/src/metabase/dashboard/constants.ts#L42

However, after testing this, it's possible that users pass a wrong theme name, which could make it behave weirdly. I don't know to what extent, but from logging, you can really set `theme` to anything. At glance, it still looks fine though.

I don't think this is an issue since `theme` isn't something users should be typing manually anyway.
![image](https://github.com/metabase/metabase/assets/1937582/869f6b14-da55-4c9b-813d-7d1f4de280a9)

Reply:
so what do you think about removing this `useEffect` block entirely?

Reply:
Removed in https://github.com/metabase/metabase/pull/44275/commits/5690e2ccf07a60d40e565440fbfdbdd597bdee02 as discussed on Slack